templates/aws/template-variables.rc

```diff
@@ -0,0 +1,16 @@
+# Kubernetes cluster configuration
+export KUBERNETES_VERSION=v1.30.0
+export CONTROL_PLANE_MACHINE_COUNT=1
+export WORKER_MACHINE_COUNT=1
+
+# AWS region
+export AWS_REGION="eu-central-1"
```

Comment:
unset default value, add a comment, like we did with vsphere `  # e.g. "eu-central-1"`